lib/revisionGuardStore/databases/mongodb.js

```diff
@@ -162,17 +162,21 @@ _.extend(Mongo.prototype, {
     this.db.close(callback || function () {});
   },
 
-  getNewId: function(callback) {
+  getNewId: function(prefix, callback) {
     callback(null, new ObjectID().toString());
   },
 
-  get: function (id, callback) {
+  get: function (prefix, id, callback) {
```

Comment:
i.e. Instead of this...
keep the old signature
```js
get: function (id, callback) {
  // and within the function
  this.options.prefix
  // ...
}
```

Reply:
it made the untit testing easier because we don't have to connect again to test the prefix itself more isolated.

also, it makes it more agnostic towards the denormalizer functionality. e.g. when we want to add a sharding feature, one denormalizer might need to handle multiple prefixes. also, 

I thought it is more transparent/ guessable when e.g. using the "set" function to see what kind of key will be generated. 

these were the thoughts when implementing it. 

I don't mind changing the signature back - for our current use case, both works. Just let me know if I should.

Reply:
@nanov What do you think?

Reply:
@nanov IMO having the same interface as before and working with this.options.prefix sounds more suitable for me...

Reply:
any decision so far? Else i'll just change it back to the old api and hope to get it merged soon then.